code/game/objects/items/devices/flashlight.dm

```diff
@@ -210,6 +210,7 @@
 	item_state = "lamp"
 	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/items_righthand.dmi'
+	force = 10

```

Comment:
Its just a lamp.

Reply:
it's a desk lamp with a metal base, those things are fucking heavy and IRL would probably knock someone unconscious (weight class is also Bulky)

Reply:
>Bulky

I went back to the code because I couldn't believe that. You're somehow right.

Reply:
You ever carried a lamp around buddy